src/mozanalysis/utils.py

```diff
@@ -26,3 +27,37 @@ def dedupe_columns(columns):
         d[col._jc] = col
 
     return list(d.values())
+
+
+def all_(l):
+    """Return the element-wise logical AND of `Column`s.
+
+    Think of this as a vector-friendly version of the built-in function
+    `all()`.
+
+    Args:
+        l: A list of `Column`s of booleans. Or, more generally,
+            an iterable of items to reduce over.
+
+    Returns:
+        A `Column` of booleans representing the logical AND. Or,
+            more generally, the result of the logical AND.
```

Comment:
I'm using up my pedantry budget for the month on the last day -- but in the general case it's a bitwise AND (or OR), not a logical AND.

I can fix this up later as part of turning on API docs for this page

Reply:
Wow, all this time I've been using bitwise AND/OR on booleans or 0/1 ints, thinking it was logical AND/OR for ndarrays - and have been doing so because of technical limitations in Python: `&` is overloadable but `and` is not, so best practice is to just use bitwise operations to do logical operations. Ew.